tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -242,7 +242,7 @@ function SplashPage() {
                       type="button"
                       class="usa-button"
                       id="viewACFFormInstructions"
-                      url="https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_data_reports_tan_ssp_instructions_definitions.pdf"
+                      href="https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_data_reports_tan_ssp_instructions_definitions.pdf"
```

Comment:
Cleaned these up while I was here. I noticed href is used in stead of url in similar places elsewhere in the code.